lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,79 @@
+# Gerrit
+
+## Supported Gerrit Versions
+
+All Gerrit 3.x Versions should be working.
+
+The current implementation uses the "hashtags" Feature from Gerrit and therefore needs a Gerrit version with [NoteDB](https://gerrit-review.googlesource.com/Documentation/note-db.html) backend.
+Gerrit 2.x with NoteDB (available only in 2.15 and 2.16 only) are untested but could work too.
+
+## Authentication
+
+<figure markdown>
+  ![Gerrit HTTP access token](/assets/images/gerrit-http-password.png){ loading=lazy }
+  <figcaption>First, create a HTTP access token for the Renovate account.</figcaption>
+</figure>
+
+Let Renovate use your HTTP access token by doing _one_ of the following:
+
+- Set your HTTP access token as a `password` in your `config.js` file
+- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`
+- Set your HTTP access token when you run Renovate in the CLI with `--password=`
+
+Make sure this user is allowed to assign the Code-Review label with "+2" to their own changes or "automerge" can't work.
+
+Remember to set `platform=gerrit` somewhere in your Renovate config file.
+
+## Renovate PR/Branch-Model with Gerrit and needed Permissions
+
+If you use the "Code-Review" label and want `automerge` working, then you have to enable `autoApprove=true` in your Renovate config.
+In this case Renovate will automatically add the _Code-Review_ label with the value "+2" to each created "pull-request" (Gerrit-Change).
+
+<!-- prettier-ignore -->
+!!! note
+    The login must be allowed to give +2 for the Code-Review label.
+
+The Renovate option `automergeType: "branch"` makes no sense for Gerrit, because there are no branches used.
```

Comment:
```suggestion
The Gerrit platform does not use Git branches.
This means you should _not_ set `automergeType: "branch"`, as it's useless here.
```

Reply:
> The Gerrit platform does not use Git branches.

That's not true, they don't need Branches for PRs.